whitelabelling_bar/README.md

```diff
@@ -0,0 +1,304 @@
+# 1. Plugin - Whitelabelling Bar
+
+This plugin adds a fixed bar to the bottom of the dashboard. The bar can display any text or an image or both
+depending on what is provided via plugin parameters.  
+The plugins is built to work with both Tiger/Panther and Bear backends.
+
+The main use case for this plugin is to add custom branding that is always visible on the dashboards even when
+embedded using iFrame.
+
+## How to work with Whitelabelling Bar Plugin
+
+1. Clone this repository
+2. Navigate to `whitelabelling_bar`
+3. Provide backend, workspace and dashboard in `.env`
+4. Rename `.env.secrets.template` to `.env.secrets` and provide your preferred way of authentication
+5. Install dependencies (`npm install` or `yarn install`)
+6. See instructions below how to build, deploy, and use this plugin
```

Comment:
@dremileta can we please sync this section to the internal repo too so that next time when updating the plugin we can bring the whole readme.